CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# v1.24.1 (2024-10-04)
+
+## Release Highlights
+* Update ecs-agent to 1.86.3 [bottlerocket-core-kit#168](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/168)
+* Update kmod-6.1-neuron to 2.18.12.0 [bottlerocket-core-kit#170](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/170)
+* Update kernels to 5.10.226 and 5.15.167 [bottlerocket-core-kit#177](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/177)
```

Comment:
These don't really meet the bar for a highlight in my opinion. We should aim to provide highlights for features, bugfixes that have issues related to them, etc.

Reply:
Gotcha, will remove these 2 lines